src/kixi/heimdall/service.clj

```diff
@@ -32,37 +32,46 @@
 
 (defn- make-auth-token [user auth-conf]
   (let [exp (-> (t/plus (t/now) (t/minutes 30)) (c/to-long))]
-    (jwt/sign user
-              (private-key auth-conf)
-              {:alg :rs256 :exp exp})))
+    (if user (try (jwt/sign user
```

Comment:
I actually wouldn't try/catch on this, sign should _always_ work, or else there's something fundamentally wrong with the app.